samples/templates/express-direct-auth/web-server/routes/login.js

```diff
@@ -84,23 +83,26 @@ router.post('/login', async (req, res) => {
     // Redirect back to home page
     res.redirect('/');
   } catch (error) {
-    renderError(res, {
-      template: 'login',
-      error,
-    });
+    authClient.transactionManager.clear();
```

Comment:
I would prefer to make this one as SDK internal logic (when reaching the terminal state, clear meta)

Reply:
ANY exception on login means the transaction must start over. We would not want to re-use an interaction handle or code challenge here. The exception could be thrown from the SDK or somewhere else. It can be expected or unexpected. The SDK can only handle expected errors that occur within itself.

Reply:
Like the case here, interactionHandle can be reused if only client errors are thrown. Just clear the meta can be a bit overkill (most probably no hurt).

Probably we can use a global error handler in the sample app to clear transaction meta depends on the error type.